cliquet/initialization.py

```diff
@@ -373,15 +374,45 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
+def _filter_events(callback, actions, resources):
+    def wrapped(event):
+        action = event.payload.get('action')
+        resource_name = event.payload.get('resource_name')
+        if action and action not in actions:
+            return
+        if resource_name and resources and resource_name not in resources:
+            return
+        return callback(event)
```

Comment:
> _filter_events returns wrapped, the decorated function.

Maybe the name of the function could make it clear that it's a decorator ?  (no strong opinion on this but since it's used inline )

> the return value of callback(event) is useless, so we could just remove this return 

Yeah, In that case I would be homogeneous and remove all the returns. It's a good practice to always return the same type from a function.